Cargo.toml

```diff
@@ -26,11 +26,6 @@ tempfile = "3.2.0"
 
 [workspace]
 members = [
-	"bridges/primitives/chain-kusama",
-	"bridges/primitives/chain-polkadot",
-	"bridges/primitives/chain-rococo",
-	"bridges/primitives/chain-westend",
-	"bridges/primitives/runtime",
```

Comment:
is this removal intended? (don't really know why it was added)

Reply:
Yes, it's intentional. We're not using `chain-kusama` or `chain-polkadot` (yet), and the others are pulled in by the [`rococo-runtime`](https://github.com/paritytech/polkadot/pull/2983/files#diff-d257db32f1322827de450e474842896422eb21d757b797fd16273874d70b2a1fR71-R73) so there's no reason (imo) for them to be explicitly part of the workspace.